src/platforms/hosted/serial_unix.c

```diff
@@ -97,6 +173,9 @@ bool serial_open(const bmda_cli_options_s *cl_opts, const char *serial)
 	}
 	fd = open(name, O_RDWR | O_SYNC | O_NOCTTY);
 	if (fd < 0) {
+		if (try_opening_network_device(name)) {
+			return true;
+		}
```

Comment:
Same comment about single-statement `if`'s, and likewise for the one below this.